spec/bowling_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'bowling'
+describe Bowling do
+  it 'it responds to add_roll' do
+    expect(subject).to respond_to :add_roll
+  end
```

Comment:
You can remove this test. As soon as you test `add_roll`, this is redundant. I would typically skip it at your level.

Reply:
Thanks, this is another habit that I've gotten into, should be fairly easy to stop doing that.